src/views/AnalyticalFramework/FrameworkDetails/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+    isNotDefined,
+} from '@togglecorp/fujs';
+import { notifyOnFailure } from '#utils/requestNotify';
+import { useRequest } from '#utils/request';
+import { AnalyticalFramework } from '#typings';
+import _ts from '#ts';
+
+import FrameworkDetailsForm from './FrameworkDetailsForm';
+import styles from './styles.scss';
+
+interface Props {
+    frameworkId: number;
+}
+
+function FrameworkDetails(props: Props) {
+    const {
+        frameworkId,
+    } = props;
+
+    const {
+        pending: frameworkGetPending,
+        response: analyticalFramework,
+    } = useRequest<AnalyticalFramework>({
+        skip: isNotDefined(frameworkId),
+        url: `server://analysis-frameworks/${frameworkId}/`,
+        method: 'GET',
+        onFailure: (_, errorBody) =>
+            notifyOnFailure(_ts('analyticalFramework', 'title'))({ error: errorBody }),
+    });
```

Comment:
Lets fetch this in AnalyticalFramework view as its values will be used in other tabs as well.
If we do that, we can directly place the Form component outside and we can remove one level of nesting.

Reply:
Let's do this when it is necessary. We need to add Framework's users table too in a later phase so the nesting of Form component makes sense here.